rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,139 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# HCA DCP Data Citation Plan
+
+## Summary
+
+Data Contributors and Data Consumers need to be able to cite data in the DCP that is referenced in scientific publications.
```

Comment:
This seems like the least important use case, if there is already a scientific publication most scientists will reference the publication and not directly link to the DCP

Reply:
Should I rephrase to state that authors of publications need to provide a citation to data in the DCP?